notebooks/tutorials/model-auditing/colab/01-user-log.ipynb

```diff
@@ -529,6 +529,16 @@
     "request"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0866c4ea",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "request.deny(reason=\"QWFEGWRTNG\")"
```

Comment:
I think we should skip this in the model auditing notebook.

Reply:
yes sry, this was not intended